src/Razor/src/Microsoft.VisualStudio.Editor.Razor/RazorIndentationFacts.cs

```diff
@@ -2,17 +2,14 @@
 // Licensed under the MIT license. See License.txt in the project root for license information.
 
 using System;
-using System.ComponentModel.Composition;
 using Microsoft.AspNetCore.Razor.Language;
 using Microsoft.AspNetCore.Razor.Language.Legacy;
 using Microsoft.AspNetCore.Razor.Language.Syntax;
 using Microsoft.VisualStudio.Text;
 
 namespace Microsoft.VisualStudio.Editor.Razor;
 
-[System.Composition.Shared]
-[Export(typeof(RazorIndentationFactsService))]
-internal class DefaultRazorIndentationFactsService : RazorIndentationFactsService
+public static class RazorIndentationFacts
```

Comment:
Is this type used by the legacy editor, or the new editor?

Reply:
It's just used by the legacy editor. FWIW, I was thinking about that and I'm going to go ahead and remove all of the breaking changes in this PR. That way, we can stage the Web Tools adoption of the shim better. So, this is going to change a bit. Stay tuned!